src/components/Menu/MenuGridItem/useMenuGridItemNavContext.tsx

```diff
@@ -10,6 +11,14 @@ export const useMenuGridItemNavContext = ({
   activeItemIndex,
   isUnderSubMenu,
   closeMenu
+}: {
+  wrapperRef?: (node: HTMLElement) => void;
+  setActiveItemIndex?: (index: number) => void;
+  getNextSelectableIndex?: (activeItemIndex: number) => number;
+  getPreviousSelectableIndex?: (activeItemIndex: number) => number;
+  activeItemIndex?: number;
+  isUnderSubMenu?: boolean;
+  closeMenu?: (option: CloseMenuOption) => void;
```

Comment:
```suggestion
  closeMenu?: (options: CloseMenuOption) => void;
```

Also, aren't these optional? If so:

```suggestion
  closeMenu?: (options?: CloseMenuOption) => void;
```